src/plotman/reporting.py

```diff
@@ -122,8 +126,23 @@ def status_report(jobs, width, height=None, tmp_prefix='', dst_prefix=''):
 
     tab.set_max_width(width)
     tab.set_deco(0)  # No borders
-    # return ('tmp dir prefix: %s ; dst dir prefix: %s\n' % (tmp_prefix, dst_prefix)
-    return tab.draw()
+
+    result = tab.draw()
+
+    # Add some summarized info
+    summary = [
+        '\n',
+        'Total jobs: {0}'.format(len(jobs))
+    ]
+
+    # Number of jobs in each tmp disk
+    tmp_dir_paths = sorted(list(map(lambda job: abbr_path(job.tmpdir, tmp_prefix), jobs)))
```

Comment:
Maybe a little tidier?

```suggestion
    tmp_dir_paths = sorted([abbr_path(job.tmpdir, tmp_prefix) for job in jobs])
```